sdk/src/main/java/io/dapr/client/DaprHttp.java

```diff
@@ -258,6 +259,12 @@ private Response doInvokeApi(String method,
     } else {
       requestBuilder.method(method, body);
     }
+
+    String daprApiToken = Properties.getStringOrDefault(Constants.DAPR_API_TOKEN, Constants.DAPR_API_TOKEN, null);
```

Comment:
Do not use the same key for Java property and Env Variable. The naming convention is different. Instead, add it directly to the Properties class as a new public static field. I recommend to refactor all Properties.get*OrDefault() to private.

Reply:
Makes sense, added it as property.